plugins/pdr_deterministic_plugin/.ci/ci_matrix.yaml

```diff
@@ -16,22 +16,83 @@ kubernetes:
 volumes:
   - {mountPath: /var/run/docker.sock, hostPath: /var/run/docker.sock}
   - {mountPath: /auto/UFM, hostPath: /auto/UFM }
+   
+runs_on_agents:
+  - nodeLabel: 'UFM-POC'
 
-
-runs_on_dockers:
-   - {file: '.ci/Dockerfile', arch: 'x86_64', name: 'plugin_worker', tag: 'latest'}
+credentials:
+  - {credentialsId: '2c8fd3fe-df05-4f6f-9d1f-f1896d611434', usernameVariable: 'REST_USER', passwordVariable: 'PASSWORD'}
 
 
 steps:
   - name: Build Plugin
-    containerSelector: "{name: 'plugin_worker'}"
+    agentSelector: "{nodeLabel: 'UFM-POC'}"
     run: |
       cd plugins/$plugin_dir/build
       bash -x ./docker_build.sh latest /
       ls -l /
       cp /ufm-plugin* /auto/UFM/tmp/${JOB_NAME}/${BUILD_ID}/
     parallel: true
 
+
+  - name: remove plugin if exists on the location
+    agentSelector: "{nodeLabel: 'UFM-POC'}"
+    run: |
+      env
+      bash -x plugins/pdr_deterministic_plugin/.ci/do_remove_plugin.sh
+    parallel: false
+    
+
+  - name: Install the new plugin on Server
+    agentSelector: "{nodeLabel: 'UFM-POC'}"
+    credentialsId: '2c8fd3fe-df05-4f6f-9d1f-f1896d611434'
+    run: |
+      env
+      pwd
+      export PASSWORD=$PASSWORD
+      bash -x plugins/pdr_deterministic_plugin/.ci/do_install_plugin_server.sh
+    parallel: false
+
+  
+  - name: load plugin
+    agentSelector: "{nodeLabel: 'UFM-POC'}"
+    run: |
+      service opensmd start
```

Comment:
what is the relationship between UFM-POC (`buleforce-031`) and `ufm-appliance04` ?
why do you start opensm on UFM-POC?

Reply:
I have no idea who this agent selector is. you have put it there..

Reply:
so there is no need to start opensmd

Reply:
pushed a comment to remove it